python/doc/examples/reliability_sensitivity/reliability/plot_crossentropy.py

```diff
@@ -0,0 +1,286 @@
+# WARNING : native parameters of distribution have to be considered
+bounds = ot.Interval([14, 500],
+                     [16, 1000])
+initial_theta = [15, 750]
+myPhysicalSpaceIS_2 = ot.PhysicalSpaceCrossEntropyImportanceSampling(event,
+                                                                     active_parameters,
+                                                                     initial_theta,
+                                                                     bounds,
+                                                                     aux_distribution)
+myPhysicalSpaceIS_2.run()
+myPhysicalSpaceISResult_2 = myPhysicalSpaceIS_2.getResult()
+print('Probability of failure:', myPhysicalSpaceISResult_2.getProbabilityEstimate())
+print('Coefficient of variation:', myPhysicalSpaceISResult_2.getCoefficientOfVariation())
+
+# %%
+# Let analyze the auxiliary output samples for the two previous simulations. We can plot initial (in blue) and auxiliary samples in physical space (orange for the first simulation and black for the second simulation).
+
+# %%
+myGraph = ot.Graph("Cloud of samples and failure domain", "R", "F", True, "topright")
+AuxiliarySamples1 = ot.Cloud(myPhysicalSpaceISResult_1.getAuxiliaryInputSample(), "orange", "fsquare", "Auxiliary samples, first case")
+AuxiliarySamples2 = ot.Cloud(myPhysicalSpaceISResult_2.getAuxiliaryInputSample(), "black", "bullet", "Auxiliary samples, second case")
+myGraph.add(InitialSamples)
+myGraph.add(AuxiliarySamples1)
+myGraph.add(AuxiliarySamples2)
+myGraph.add(mycontour)
+view = viewer.View(myGraph)
+
+# %%
+# By analyzing the failure samples, one may want to include correlation parameters in the auxiliary distribution. In this last example, we add a Normal copula. The correlation parameter will be optimized with associated interval between 0 and 1.
+
+# %%
+ot.RandomGenerator.SetSeed(0)
+distribution_marginR = ot.LogNormalMuSigma(3e6, 3e5, 0.0).getDistribution()
+distribution_marginF = ot.Normal(750., 50.)
+aux_marginals = [distribution_marginR, distribution_marginF]
+copula = ot.NormalCopula()
+aux_distribution = ot.ComposedDistribution(aux_marginals, copula)
+print('Initial parameters of auxiliary distribution:', aux_distribution.getParameter())
+# Definition of parameters to be optimized
+active_parameters = [0, 1, 2, 3, 4, 5]
+bounds = ot.Interval([14, 0.01, 0., 500., 20., 0.],
+                     [16, 0.2, 0.1, 1000., 70., 1.])
+initial_theta = distribution.getParameter()
+myPhysicalSpaceIS_3 = ot.PhysicalSpaceCrossEntropyImportanceSampling(event,
+                                                                     active_parameters,
+                                                                     initial_theta,
+                                                                     bounds,
+                                                                     aux_distribution)
+myPhysicalSpaceIS_3.run()
+myPhysicalSpaceISResult_3 = myPhysicalSpaceIS_3.getResult()
```

Comment:
could we avoid variables names with myXXX ? 
just algo3, result3, this is simpler

Reply:
Thanks, this is now fixed.